api/app/Models/User.php

```diff
@@ -667,4 +673,34 @@ public function getIsIndigenousAttribute() {
 
         return null; // if indigenousCommunities is null then so is isIndigenous
     }
+
+    /* accessor to maintain functionality of to be deprecated languageAbility field, its logic comes from migration drop_language_ability*/
+    public function getLanguageAbilityAttribute($languageAbility = null) {
+        // if the field exists, say for migration purposes, must stop accessor overriding
+        if($languageAbility !== null){
+            return $languageAbility;
+        }
```

Comment:
Ahhh, I see. Actually, this highlights why its generally a bad idea to rely on the Eloquent class in a database migration. The class can change in ways that doesn't reflect the state of the database.